bin/www

```diff
@@ -8,6 +8,13 @@ var app = require('../server/index');
 var debug = require('debug')('src:server');
 var http = require('http');
 
+const mongoose = require('mongoose')
+mongoose.Promise = global.Promise
+const connection = 'mongodb://localhost/lizardboard'
+
+mongoose.connect( process.env. MONGODB_URI )
+  .then( () => server.listen( port ) )
+server.listen( port );
```

Comment:
Remove this line - no need to start the server twice.  I think the `server.on` lines also need to be above this to make sure set up occurs properly